app/views/internal/path_redirects/index.html.erb

```diff
@@ -0,0 +1,56 @@
+<div class="row m-3">
+  <div class="col">
+    <a href="<%= new_internal_path_redirect_path %>" class="btn btn-primary" role="button">Create a New Path Redirect</a>
+  </div>
+  <div class="col">
+    <%= search_form_for @q, url: internal_path_redirects_path, class: "form-inline justify-content-end" do |f| %>
+
+      <%= f.label :old_path_or_new_path_cont, "Old or new path", class: "sr-only" %>
+      <%= f.search_field :old_path_or_new_path_cont, placeholder: "Old or new path", class: "form-control mx-3" %>
```

Comment:
This is not a blocker by any means and is just me being nit-picky, but I would prefer the placeholder to read "Old or New Path" rather than what it currently is--I think it looks slightly weird as is locally. 😅

Reply:
Updated!